cmd/ak/cmd/manifest/apply.go

```diff
@@ -2,100 +2,45 @@ package manifest
 
 import (
 	"fmt"
-	"io"
 	"os"
-	"strings"
 
 	"github.com/spf13/cobra"
-	"github.com/xeipuuv/gojsonschema"
-	"gopkg.in/yaml.v3"
 
-	"go.autokitteh.dev/autokitteh/backend/apply"
 	"go.autokitteh.dev/autokitteh/cmd/ak/common"
-	"go.autokitteh.dev/autokitteh/internal/kittehs"
+	"go.autokitteh.dev/autokitteh/sdk/sdkmanifest"
 )
 
-type output struct {
-	Logs       any `json:"logs"`
-	Operations any `json:"operations"`
-}
-
-var dryRun, noValidate bool
+var dryRun bool
 
 var applyCmd = common.StandardCommand(&cobra.Command{
-	Use:     "apply [file] [--no-validate] [--dry-run]",
+	Use:     "apply [file] [--no-validate] [--from-scratch] [--dry-run]",
```

Comment:
You split the planning into a separate command - why? Anyway, `--dry-run` (i.e. plan without execute) has no meaning or usage now.

Reply:
discussed in https://github.com/autokitteh/autokitteh/pull/6#discussion_r1496787189